From 84e559409a99eea7f7b2ad90c16293f356c05c0f Mon Sep 17 00:00:00 2001
From: Yongli He <yongli.he@windriver.com>
Date: Thu, 25 Aug 2011 16:38:13 +0800
Subject: [PATCH] fsl_p50x0: Fix 8250/16550 to handle IRQ storm after receipt of a break

FSL 8250 serial suffer this problem on both PPC32 and PPC64.

Signed-off-by: Yongli He <yongli.he@windriver.com>
---
 drivers/serial/Kconfig |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/serial/Kconfig b/drivers/serial/Kconfig
index 902d82c..158c7c1 100644
--- a/drivers/serial/Kconfig
+++ b/drivers/serial/Kconfig
@@ -72,7 +72,7 @@ config SERIAL_8250_CONSOLE
 
 config SERIAL_8250_PPC_BUG
 	bool "Fix 8250/16550 to handle IRQ storm after receipt of a break"
-	depends on SERIAL_8250 && PPC32
+	depends on SERIAL_8250 && PPC
 	---help---
 	  If you say Y here, additional checks will be added to handling of
 	  interrupts on the serial ports which will prevent ill effects of
-- 
1.7.0.4

